src/content/getting-started/quickstart/managed-kubernetes/iks/_index.md

```diff
@@ -0,0 +1,220 @@
+| secondary    | 172.17.64.0/18 | 172.22.0.0/16 |
+
+### Submariner deployment
+
+#### subctl binary
+
+{{% subctl-install %}}
+
+#### Access
+
+At the time of writing, `subctl` does not support OIDC-based authentication,
+so accessing the clusters can be done through [Service Accounts](https://kubernetes.io/docs/concepts/security/service-accounts/).
+
+The fastest way to obtain a suitable kube config is to run the following command:
+
+```shell
+ibmcloud ks cluster config --admin --cluster CLUSTER_NAME
+```
+
+The example uses `$primary_ctx` and `$secondary_ctx` environment variables.
+Their values can be obtained using `kubectl` like:
+
+```shell
+kubectl config get-contexts -o name
+export primary_ctx=CLUSTER1_NAME
+export secondary_ctx=CLUSTER2_NAME
+```
+
+#### Calico CNI
+
+Due to [active-passive gateway model](https://submariner.io/getting-started/architecture/gateway-engine/)
+of Submariner, "external" (cross-cluster) traffic will appear on the nodes.
+As a consequence, in-cluster node-to-node traffic encapsulation should be configured.
+
+```shell
+kubectl patch --type=merge IPPool default-ipv4-ippool -p '{"spec":{"ipipMode":"Always"}}'
+```
+
+On an IBM Cloud Kubernetes (and OpenShift) cluster, the default encapsulation mode is IP in IP,
+which is acceleration-assisted, so it is recommended to keep that.
+Avoiding the `Always` encapsulation is also an option, but it requires configuring reverse path filtering on the nodes.
+
+Calico should be also configured to avoid unnecessary NAT on cross-cluster traffic.
+Please read the [Calico specific](https://submariner.io/operations/deployment/calico/) guide on how to set it up.
+
+Every Submariner cluster should have something like this:
+
+```yaml
+# see: https://submariner.io/operations/deployment/calico/
+# applicable on the first cluster
+---
+apiVersion: crd.projectcalico.org/v1
+kind: IPPool
+metadata:
+  name: avoid-nat-towards-cluster2-pods
+spec:
+  cidr: 172.17.64.0/18  # secondary cluster's pod subnet
+  natOutgoing: false    # what we want
+  disabled: true        # Calico should not use the mentioned subnet to assign IPs from
+---
+apiVersion: crd.projectcalico.org/v1
+kind: IPPool
+metadata:
+  name: avoid-nat-towards-cluster2-services
+spec:
+  cidr: 172.22.0.0/16
+  natOutgoing: false
+  disabled: true
+```
+
+### Broker
+
+Deploy the [Broker](https://submariner.io/getting-started/architecture/broker/) on the primary cluster.
+
+``` bash
+subctl deploy-broker --context $primary_ctx
+```
+
+The command will output a file named `broker-info.subm` to the directory it is run from, which will be used to set up the
+IPsec tunnel between clusters.
+
+#### Join clusters
+
+Now it is time to register every cluster in the ClusterSet of the Broker.
+
+First, join the Broker-hosting cluster itself to the Broker:
+
+```shell
+./subctl join --context $primary_ctx \
+  broker-info.subm \
+  --clusterid my-cluster-name1 \
+  --load-balancer \
+  --clustercidr "172.17.0.0/18"
```

Comment:
Normally the ClusterCIDRs should be auto-discovered by the submariner-operator and we really dont need to explicitly specify them. Have you seen any issues in the auto-discovery code and hence passing the `--clustercidr` here?

Reply:
Good point @sridhargaddam, the operator does not find the ClusterCIDR. So I opted to manually sepcify it.
I checked the relevant operator code and I would contribute the detection failure to the missing kube-proxy deployment. It is managed by a systemd service currently on IKS nodes.
My best guess is that the ClusterCIDR information could be obtained from `kubectl get ippools.crd.projectcalico.org default-ipv4-ippool -ojson | jq '.spec.cidr'`